app/templates/application.hbs

```diff
@@ -13,8 +11,18 @@
       <nav.item>
         <nav.link-to @route="sparql">SPARQL</nav.link-to>
       </nav.item>
+    {{#if this.session.isAuthenticated}}
+      <BsButton
```

Comment:
Is there a specific reason why you don't use the `MuLogout` component provided by the addon?  (or create a new mu-logout component inheriting from the one in ember-mu-login)